src/xunit.performance.api/ScenarioInfo.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.Diagnostics.Tracing.Session;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace Microsoft.Xunit.Performance.Api
+{
+    public sealed class ScenarioInfo
```

Comment:
This should have a more clear name - the name should clarify when it is provided and what it is for.  "ScenarioExecutionResult" or something like that.  You get one per iteration, right?  So you could imagine also including Iteration in the name so that you can get a collection of them for a given run of a scenario.